backend/src/akvo/lumen/endpoint/aggregation.clj

```diff
@@ -23,7 +24,16 @@
                              visualisation-type
                              (json/parse-string query keyword))
           (catch JsonParseException e
-            (http/bad-request {:message (.getMessage e)})))
+            (log/error e)
+            (http/bad-request
+             {:message (format "JSON parse exception: %s" (.getMessage e))}))
+          (catch java.sql.SQLException e
+            (log/error e)
+            (http/bad-request
+             {:message  (format "Database query exception: %s" (.getMessage e))}))
```

Comment:
I guess we expose this message to the user? I don't think is _useful_ to export database errors but meaningful ones